Rules/CommonScripts/ChatCommands/BlobCommands.as

```diff
@@ -0,0 +1,178 @@
+#include "ChatCommand.as"
+#include "MakeSeed.as";
+#include "MakeCrate.as";
+#include "MakeScroll.as"
+
+class PineTreeCommand : ChatCommand
+{
+	PineTreeCommand()
+	{
+		super("pinetree", "Spawn a pine tree seed");
+		AddAlias("pineseed");
+		AddAlias("tree");
+		AddAlias("seed");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
+
+		CBlob@ blob = player.getBlob();
+		if (blob !is null)
+		{
+			Vec2f pos = blob.getPosition();
+			server_MakeSeed(pos, "tree_pine", 600, 1, 16);
+		}
+		else
+		{
+			server_AddToChat(getTranslatedString("Blobs cannot be spawned while dead or spectating"), ConsoleColour::ERROR, player);
+		}
+	}
+}
+
+class BushyTreeCommand : ChatCommand
+{
+	BushyTreeCommand()
+	{
+		super("bushytree", "Spawn a bushy tree seed");
+		AddAlias("bushyseed");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
+
+		CBlob@ blob = player.getBlob();
+		if (blob !is null)
+		{
+			Vec2f pos = blob.getPosition();
+			server_MakeSeed(pos, "tree_bushy", 400, 2, 16);
+		}
+		else
+		{
+			server_AddToChat(getTranslatedString("Blobs cannot be spawned while dead or spectating"), ConsoleColour::ERROR, player);
+		}
+	}
+}
+
+class CrateCommand : ChatCommand
+{
+	CrateCommand()
+	{
+		super("crate", "Spawn a crate with an optional blob inside");
+		SetUsage("[blob] [description]");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
+
+		CBlob@ blob = player.getBlob();
+		if (blob is null)
+		{
+			server_AddToChat(getTranslatedString("Blobs cannot be spawned while dead or spectating"), ConsoleColour::ERROR, player);
```

Comment:
This also duplicates some of the blob spawning code but w/e on the more complex ones